src/components/Sidebar.js

```diff
@@ -77,17 +77,25 @@ class Sidebar extends Component {
         <div className="main-panel">
 
           <NavTop />
-          <Switch>
-            <Route path={`/${localStorage.getItem('username')}`} render={() => <Dashboard />}/>
-            <Route path={`/${localStorage.getItem('username')}/rubrics`} render={() => <RubricsIndex />}/>
-          </Switch>
-          <Footer />
 
+          <div className="content">
+            <div className="container-fluid">
+
+              <Switch>
+                <Route path={`/:username/dashboard`} component={Dashboard} />
+                <Route path={`/:username/rubrics`} component={RubricsIndex} />
+              </Switch>
+              {/*<Route path={`/${localStorage.getItem('username')}`} render={() => <Dashboard />}/>
```

Comment:
Do we still need the old codes that are commented out for Route Path stored in local storage?

Reply:
Probably not.